curated_transformers/models/hf_wrapper.py

```diff
@@ -1,29 +1,27 @@
-import re
-import torch
-
-from typing import Tuple, List, Dict
+from typing import Tuple, List, Dict, Callable
 from functools import partial
 
 from spacy.tokens import Doc
 from thinc.api import Model, torch2xp, get_current_ops
 from thinc.api import xp2torch, PyTorchWrapper_v2
+from thinc.model import empty_init
 from thinc.shims.pytorch_grad_scaler import PyTorchGradScaler
 from thinc.types import ArgsKwargs, Floats2d, Ints1d, Ragged
 
 from .bert import BertEncoder, BertConfig
 from .roberta import RobertaEncoder, RobertaConfig
 from .._compat import transformers, has_hf_transformers
+from ..util import registry
```

Comment:
Seems like the `util` module is not part of this changeset?

Reply:
Oops, added.